src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +97,7 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={!isAddButtonDisabled}
```

Comment:
update this condition to get button state to be clearer. 
Because in this case `!isAddButtonDisabled` it equal to `isAddButtonAbled`, so how `disabled` can be set when button is abled 
```suggestion
            disabled={isAddButtonDisabled}
```